docs/source/release/v6.4.0/Workbench/Bugfixes/33663.rst

```diff
@@ -0,0 +1 @@
+- Fixed where matplotlib 3.5 didn't allow for waterfall plots to functions
```

Comment:
I think it fixed a bunch of matplotlib depreceation problems, not just waterfall plots

Reply:
We already have most of the issues described in the other release notes made by matplotlib work for workbench